pkgs/development/libraries/taco/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, fetchgit
+, cmake
+, enablePython ? false
+, python
```

Comment:
Yes, usually `null` is an antipattern, but I think here it is ok. The problem is caused by the normal package, not the python package. Since the normal package lives outside the python subset setting it to null here seems to be necessary. I don't fully understand why this is the case. It seems to be a preferred solution by looking at similar packages.

Reply:
OK, I've pushed it. Let's see if ofborg is happy.